checkstyle-tester/diff.groovy

```diff
@@ -625,11 +626,11 @@ def getProjectsStatistic(diffDir) {
 def runMavenExecution(srcDir, excludes, checkstyleConfig,
                       checkstyleVersion, extraMvnRegressionOptions) {
     println "Running 'mvn clean' on $srcDir ..."
-    def mvnClean = "mvn --batch-mode clean"
+    def mvnClean = "mvn -e --no-transfer-progress --batch-mode clean"
     executeCmd(mvnClean)
     println "Running Checkstyle on $srcDir ... with excludes {$excludes}"
-    def mvnSite = "mvn -e --batch-mode site -Dcheckstyle.config.location=$checkstyleConfig " +
-        "-Dcheckstyle.excludes=$excludes"
+    def mvnSite = "mvn -e --no-transfer-progress --batch-mode site "
+        + "-Dcheckstyle.config.location=$checkstyleConfig -Dcheckstyle.excludes=$excludes"
```

Comment:
https://app.travis-ci.com/github/checkstyle/contribution/jobs/590806173#L567
Looks like CI failed on this. Am I blind as I am not seeing an issue besides the `+` is in a different location than before?

Reply:
Local run:
````
Running Checkstyle on src\main\java ... with excludes {}
Caught: groovy.lang.MissingMethodException: No signature of method: java.lang.String.positive() is applicable for argument types: () values: []
Possible solutions: notify(), size(), tokenize(), size()
groovy.lang.MissingMethodException: No signature of method: java.lang.String.positive() is applicable for argument types: () values: []
Possible solutions: notify(), size(), tokenize(), size()
        at diff.runMavenExecution(diff.groovy:633)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at diff$_generateCheckstyleReport_closure4.doCall(diff.groovy:292)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at diff.generateCheckstyleReport(diff.groovy:265)
        at diff$generateCheckstyleReport$10.callCurrent(Unknown Source)
        at diff.launchCheckstyleReport(diff.groovy:226)
        at diff$launchCheckstyleReport$6.callCurrent(Unknown Source)
        at diff.run(diff.groovy:31)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
````